gnmi_server/server_test.go

```diff
@@ -134,6 +134,25 @@ func createReadServer(t *testing.T, port int64) *Server {
 	return s
 }
 
+func createRejectServer(t *testing.T, port int64) *Server {
+	certificate, err := testcert.NewCert()
+	if err != nil {
+		t.Errorf("could not load server key pair: %s", err)
+	}
+	tlsCfg := &tls.Config{
+		ClientAuth:   tls.RequestClientCert,
+		Certificates: []tls.Certificate{certificate},
+	}
+
+	opts := []grpc.ServerOption{grpc.Creds(credentials.NewTLS(tlsCfg))}
+	cfg := &Config{Port: port, EnableTranslibWrite: true,  Threshold: -1}
```

Comment:
This test will reject any subscribe request.
I'd like to use bigger threshold for test, for example, GNMI accepts the first request, and reject the second request.

Reply:
Set threshold to 2, so will accept first 2 subscribe requests and rejects the third.